src/prod/src/Hosting2/README.md

```diff
@@ -0,0 +1,4 @@
+# Hosting subsystem
+
+The [Cluster Manager](https://github.com/Microsoft/service-fabric/tree/f258f7579af9643dac6b1c75c93db9a3bcd28fdd/src/prod/src/Management/ClusterManager) informs the **Hosting subsystem** (running on each node) which services it needs to manage for a particular node. The Hosting subsystem then manages the lifecycle of the application on that node. It interacts with the [Reliability](https://github.com/Microsoft/service-fabric/tree/master/src/prod/src/Reliability#reliability-subsystem) and [Health](https://github.com/Microsoft/service-fabric/tree/master/src/prod/src/Management/healthmanager) services to ensure that the replicas are properly placed, healthy, and happy.  
```

Comment:
replace with relative links so you aren't pointing to a specific commit for link to Cluster Manager